src/test/java/com/lmax/disruptor/dsl/DisruptorTest.java

```diff
@@ -86,6 +86,21 @@ public void tearDown() throws Exception
         executor.joinAllThreads();
     }
 
+    @Test
+    public void shouldHaveStartedAfterStartCalled() throws Exception {
+      if (disruptor.hasStarted())
+      {
+        fail("Should only be set to started after start is called");
+      }
+
+      disruptor.start();
+
+      if (!disruptor.hasStarted())
```

Comment:
Using `if` and `fail` is an _interesting_ way of writing this test.

Would it not be simpler use `assertTrue(disruptor.hasStarted(), "Should be set to started after start is called");`?